### PR TITLE
add doc page for pipeline

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

File: source/docs/pipeline_and_webpack.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* Importing CSS, SCSS & Fonts in JS
+* Support for React & JSX
+
+### Ejecting & Customizing the Webpack config
+
+If you want to configure Embark's webpack for your own needs you can run:
+
+<pre><code class="shell">$ embark eject-webpack</code></pre>
+
+This will output the webpack file to `webpack.config.js`, you can then customize this file to suit your needs.
```
Comment (andremedeiros) [inline source/docs/pipeline_and_webpack.md]: This command outputs the webpack file named `webpack.config.js` to the root directory of your dApp. You can then customize this to suit your specific needs.

File: source/docs/pipeline_and_webpack.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<pre><code class="shell">$ embark run --pipeline production</code></pre>
+
+### Default webpack config
+
+Embark has a internal default webpack config, this config allows you out of the box to:
+
+* ES6 modules & imports
+* Importing CSS, SCSS & Fonts in JS
+* Support for React & JSX
+
```
Comment (michaelsbradleyjr) [inline source/docs/pipeline_and_webpack.md]: Could add...

+ Automatic support for older browsers via babel's `[preset-env](https://babeljs.io/docs/en/babel-preset-env)`. Embark's default [Browserlist](https://github.com/browserslist/browserslist) setting is `['last 1 version', 'not dead', '> 0.2%']`.

File: source/docs/pipeline_and_webpack.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+You can also specify a particular mode with `--pipeline`, for e.g
+
+<pre><code class="shell">$ embark run --pipeline production</code></pre>
+
+### Default webpack config
+
+Embark has a internal default webpack config, this config allows you out of the box to:
+
+* ES6 modules & imports
```
[the remark this answers is not in the document]
Comment (iurimatias) [reply source/docs/pipeline_and_webpack.md]: does es6 modules implies the imports are supported?

[the remark this answers is not in the document]
Comment (michaelsbradleyjr) [reply source/docs/pipeline_and_webpack.md]: I would interpret "ES6 modules" to be shorthand for "support for ES6 module syntax for imports and exports".

**Pull request merged**